doc/Caliptra.md

```diff
@@ -415,21 +415,181 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
+9. HVM, through JTAG or using the SoC interface, polls for “requested service\[s\]” bit\[s\] available in “CPTRA\_BOOT\_STATUS” register.
+10. HVM, through JTAG, reads mbox\_status\[3:0\] to check if the data is ready to be read (DATA\_READY encoding).
+11. HVM must write a bit in CPTRA\_DBG\_MANUF\_SERVICE\_REG over JTAG, indicating that it completed reading the CSR.
+12. Caliptra IP HW opens the Caliptra Mailbox for SoC usages, such as FW loading (if required in some HVM flows). Until this write operation is complete, the SoC does not get a grant/lock of the APB-exposed mailbox interface.
 
 ## Certificate format
-
-Device identity certificates follow the X.509 v3 format described in RFC 5280.  The values in the X.509 certificate follow the DICE TCBInfo fields, as defined in [Reference 5](#ref-5). The owner public key is extended into VendorInfo, with the security operational state reflecting the flags of DICE TCBInfo.  Additional fields may be extended into VendorInfo.
+Caliptra certificates follow the X.509 v3 format described in RFC 5280. After vendor provisioning, Caliptra's certificate chain contains the following certificates:
+1. Provisioner CA (may be one or more certificates)
+2. IDevID
+3. LDevID
+4. Alias<sub>FMC</sub>
+5. Alias<sub>RT</sub>
+6. DPE
+
+After owner provisioning, an Owner CA may endorse the IDevID, LDevID, or Alias<sub>FMC</sub> public keys. Owner CA provisioning is outside the scope of this specification.
+
+Caliptra generates the LDevID, Alias<sub>FMC</sub>, Alias<sub>RT</sub>, and DPE certificates. The vendor and optionally the owner generate all other certificates.
+
+### Provisioner CA
+Provisioner CA (pCA) is a set of one or more certificates issued by the vendor. The vendor is responsible for provisioning pCA to the SoC. Caliptra does not consume pCA. See [Reference 5](#ref-5) for guidance on pCA.
+
+### IDevID certificate
+The vendor issues the IDevID certificate during SoC manufacturing. As part of [provisioning IDevID during manufacturing](#provisioning-idevid-during-manufacturing), Caliptra uses the UDS to derive the IDevID key pair and generate a CSR. The vendor's pCA uses the CSR to generate and sign the IDevID certificate.
+
+For IDevID to endorse LDevID, Caliptra requires the vendor to implement the following field values for the IDevID certificate:
+
+*Table 7: IDevID certificate fields*
+
+| Field                          | Sub field    | Value
+| -------------                  | ---------    | ---------
+| Version                        | v3           | 2
+| Serial Number                  | -            | First 20 bytes of SHA256 hash of IDevID public key
+| Validity                       | notAfter     | 99991231235959Z
+| Subject Name                   | CN           | Caliptra IDevID
+|                                | serialNumber | SHA384 hash of Subject public key
```

Comment:
1. Hex encoded printable string
2. We could be clear about the format of the hashed key. I think we use "uncompressed" form